src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs

```diff
@@ -732,9 +743,13 @@ internal void ReportMarkedAsRemovedButNotActuallyRemovedApiList(Action<Diagnosti
 
             private bool IsTrackedAPI(ISymbol symbol, CancellationToken cancellationToken)
             {
-                if (symbol is IMethodSymbol methodSymbol && s_ignorableMethodKinds.Contains(methodSymbol.MethodKind))
+                if (symbol is IMethodSymbol methodSymbol)
                 {
-                    return false;
+                    if (s_ignorableMethodKinds.Contains(methodSymbol.MethodKind))
+                        return false;
+
+                    if (methodSymbol is { MethodKind: MethodKind.Ordinary, CanBeReferencedByName: false })
```

Comment:
❓ Shouldn't we include the `<Clone>` method? Users might not realize that converting a record back to a non-record will always be a breaking change.

Reply:
hrmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm.  ok.  that is a compellign point.